parla/task_runtime.py

```diff
@@ -936,8 +984,14 @@ def check_resources_availability(self, d: Device, resources: ResourceDict):
             is_available = True
             for name, amount in resources.items():
                 dres = self._devices[d]
+
+                # Workaround stupid vcus (I'm getting rid of these at some point)
```

Comment:
Does workaround mean it just runs with no effects?

Reply:
Yup. For now at least. None of our apps use these at the moment. I think we should worry about fixing them later, but not this week.